src/EA.Weee.Core/Organisations/SoleTraderViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace EA.Weee.Core.Organisations
+{
+    using System.ComponentModel;
+    using System.ComponentModel.DataAnnotations;
+    using EA.Weee.Core.DataStandards;
+    public class SoleTraderViewModel
+    {
+        [Required]
+        [DisplayName("First name")]
+        [StringLength(CommonMaxFieldLengths.DefaultString)]
```

Comment:
@stevelee-civica have a look at the latest partner view model as I think they use different length fields, think there is a Firstname and lastname one

Reply:
updated